src/Neo/Ledger/MemoryPool.cs

```diff
@@ -390,7 +394,8 @@ private List<Transaction> RemoveOverCapacity()
             List<Transaction> removedTransactions = new();
             do
             {
-                PoolItem minItem = GetLowestFeeTransaction(out var unsortedPool, out var sortedPool);
+                var minItem = GetLowestFeeTransaction(out var unsortedPool, out var sortedPool);
+                if (minItem == null || sortedPool == null) break;
```

Comment:
This can fail if it's null, because lime 259 and 262

Reply:
Where is the closing `}` from `line 397`. Looks like your missing one

Reply:
> This can fail if it's null, because lime 258 and 261

It doesn't matter which ever is `true` 1st. Doesn't matter even if both are `null`.